Library/Homebrew/utils/github.rb

```diff
@@ -340,6 +340,15 @@ def create_fork(repo)
     open_api(url, data: data, scopes: scopes)
   end
 
+  def check_fork_exists(repo)
+    username = api_credentials[1]
```

Comment:
```suggestion
    _, username = api_credentials
```

Reply:
is there an advantage to using this syntax?

Reply:
Yes, it will avoid failing if it's not an array.